README.md

```diff
@@ -33,6 +33,14 @@ Jane's ESLint plugin and configurations.
 ```
 
 You can extend any or all of the exported configurations.
+If you are using the typescript plugin, some of the rules require this to be
+added to the .eslintrc file
+```json
+"parser": "@typescript-eslint/parser",
```

Comment:
Is this necessary, since you added this to the config?

Reply:
i was debating whether to remove it. i think it might be necessary if there is a nested eslintrc